package.json

```diff
@@ -67,5 +67,8 @@
 		"*.{js,ts,vue}": "eslint --fix",
 		"*.md": "prettier --write",
 		"*.{css,scss,vue}": "stylelint --fix"
+	},
+	"dependencies": {
+		"cross-env": "^7.0.3"
```

Comment:
yes - agree.

Reply:
@paescuj will you do the update?  I can't because I currently can't rebase to latest version... Reason:  vue vs. vue-template-compiler dependency issue... I opened an issue for that.

Reply:
@paescuj btw - the changes in package-lock.json can be ignored / deleted...

Reply:
Sorry, can't do that! I don't have permissions on this repository.

Reply:
OK - as soon as the vue dependency issue is resolved I will do the updates...

Reply:
Done